overlord/snapstate/backend/setup.go

```diff
@@ -39,8 +39,16 @@ type InstallRecord struct {
 	TargetSnapExisted bool `json:"target-snap-existed,omitempty"`
 }
 
+type SetupSnapOpts struct {
```

Comment:
super small nitpick: I think most options I've seen in the code end with "Options" instead of "Opts" so that would be a more consistent alternative